packages/@aws-cdk/triggers/package.json

```diff
@@ -77,7 +77,7 @@
     "@aws-cdk/cdk-build-tools": "0.0.0",
     "@aws-cdk/integ-runner": "0.0.0",
     "@aws-cdk/aws-sns": "0.0.0",
-    "aws-sdk": "^2.1211.0",
+    "aws-sdk": "^2.1282.0",
```

Comment:
Why the manual update here?

Reply:
My bad, in hindsight this wasn't needed. I could not find the `httpOptions` property on the SDK clients, which did appear if I installed another version. But that is because the node_modules are in the root of the project instead of in the subproject for triggers only. 

Reverted in next commit.